src/modules/dashboard/sagas/colony.ts

```diff
@@ -175,6 +242,8 @@ function* colonyAvatarUpload({
 
     yield put<AllActions>({
       type: ActionTypes.COLONY_AVATAR_UPLOAD_SUCCESS,
+      meta,
```

Comment:
Do we still need these (meta and payload)?

I removed them since I've also removed the reducer, so nothing is processing the data that's being passed from here

Reply:
`payload` not necessarily but I'd suggest to just leave it in for simplicity's sake. `meta` we do need as it's passed through to the async function generator